setup.py

```diff
@@ -38,14 +43,10 @@
               'linter'],
     classifiers=[
         'Development Status :: 4 - Beta',
-##      'Development Status :: 5 - Production/Stable', eventually...
+        # 'Development Status :: 5 - Production/Stable', eventually...
```

Comment:
Eh, just remove the entire line, I don't think I'll ever be brave enough to call this stable.  (And if I do, I can look up the spelling.)